release.json

```
@@ -10,8 +10,22 @@
 		"INTEGRATIONS_CORE_VERSION": "master",
 		"OMNIBUS_SOFTWARE_VERSION": "master",
 		"OMNIBUS_RUBY_VERSION": "datadog-5.5.0",
-		"JMXFETCH_VERSION": "0.33.1",
-		"JMXFETCH_HASH": "140dd8568b78f5a616da8664f34bb37397c3d3ed87af106d632714f7218e8e88"
+		"JMXFETCH_VERSION": "0.34.0",
+		"JMXFETCH_HASH": "b5c053b960f9fa949a76ec3b84c308586c409f956379481e6a0044867deb7b52"
+	},
+	"7.17.0-rc.1": {
+		"INTEGRATIONS_CORE_VERSION": "7.17.0-rc.1",
+		"OMNIBUS_SOFTWARE_VERSION": "7.17.0-rc.1",
+		"OMNIBUS_RUBY_VERSION": "datadog-5.5.0",
+		"JMXFETCH_VERSION": "0.34.0",
+		"JMXFETCH_HASH": "b5c053b960f9fa949a76ec3b84c308586c409f956379481e6a0044867deb7b52"
+	},
+	"6.17.0-rc.1": {
+		"INTEGRATIONS_CORE_VERSION": "6.17.0-rc.1",
+		"OMNIBUS_SOFTWARE_VERSION": "6.17.0-rc.1",
```

```suggestion
		"OMNIBUS_SOFTWARE_VERSION": "7.17.0-rc.1",
```

Both releases can point to the same `integrations-core` version.